py3status/modules/do_not_disturb.py

```diff
@@ -1,109 +1,225 @@
 # -*- coding: utf-8 -*-
 """
-Turn on and off dbus notifications.
+Turn on and off desktop notifications.
 
-A left mouse click will toggle the state of this module.
+This module supports dunst, xfce4-notifyd, and lxqt-notificationd.
+For best results, don't install more than one notification server.
 
 Configuration parameters:
-    dunst_signal_off: Signal to send to the dunst process to disable "Do Not Disturb."
-        (default 'SIGTERM')
-    dunst_signal_on: Signal to send to the dunst process to enable "Do Not Disturb."
-        (default 'SIGUSR1')
-    format: Display format for the "Do Not Disturb" module.
-        (default '{state}')
-    notification_manager: The process name of your notification manager.
-        (default 'dunst')
-    refresh_interval: Refresh interval to use for killing notification manager process.
-        (default 0.25)
-    state_off: Message when the "Do Not Disturb" mode is disabled.
-        (default 'OFF')
-    state_on: Message when the "Do Not Disturb" mode is enabled.
-        (default 'ON')
+    cache_timeout: refresh interval for this module; for xfce4-notifyd
+        (default 10)
+    format: display format for this module
+        (default '{server} [\?if=state&color=good DND|\?color=bad DND]')
+    pause: specify whether to pause or kill process; for dunst
```

Comment:
Something like this might explain the feature

If True then notifications will be paused when  "Do Not Disturb" mode is enabled, they will be shown when "Do Not Disturb" mode is disabled.  If False then notifications during this time will be discarded.  Note: only `dunst` supports this option.

The name could be nicer too.

Reply:
I resolved this with...
```
    pause: specify whether to pause or kill process; for dunst
        see `man -P 'less +/MISCELLANEOUS' dunst` for more information
        (default False)
```

However, I wonder if we want `pause` or `paused`.

Reply:
@tobes Are we resolved here? I want to merge this today to clean up idle issues + idle PRs.

`man -P 'less +/MISCELLANEOUS' dunst` will take users to...
```
MISCELLANEOUS
       Dunst can be paused by sending a notification with a summary of "DUNST_COMMAND_PAUSE" and resumed with a
       summary of "DUNST_COMMAND_RESUME".  Alternatively you can send SIGUSR1 and SIGUSR2 to pause and unpause
       respectively. For Example:

       killall -SIGUSR1 dunst # pause
       killall -SIGUSR2 dunst # resume

       When paused dunst will not display any notifications but keep all notifications in a queue.  This can for
       example be wrapped around a screen locker (i3lock, slock) to prevent flickering of notifications through
       the lock and to read all missed notifications after returning to the computer.

```

Reply:
The idea is that the module documentation explains stuff not starts a treasure hunt.  Is this such a difficult concept?

Also for me the documentation is more important than the code because if you cannot document it well then how can I trust the code?

Reply:
I enjoy treasure hunts. We should plan a fun treasure hunt one day. :skull_and_crossbones: 

 I removed `do_not_disturb: new module takeover` commit too as it is difficulty to always maintain/upkeep the last commit. When I change something, I need to do it in first commit, then I have to deal with second commit too because this moves the file to new location. Repeat hassles.

Trust this answers your query.

Reply:
>Trust this answers your query. 

?

Reply:
No it does not even come close :(

Reply:
We need to peel off that extra layer of crazy off you. Give me diff and I'll apply it. We may have to remove duplicate `@author` because I think it could be applied to the last one (ie me). \o/